setup/express.js

```diff
@@ -0,0 +1,122 @@
+const expressWinston = require('express-winston');
+
+const { setupNodeLogger } = require('./winston');
+const { DEFAULT_LOG_LEVEL } = require('..');
```

Comment:
It creates the circular dependency, which should be rather avoided.
Can we just extract the DEFAULT_LOG_LEVEL to constsor some other file?

Reply:
Yeah, I'd use some `constants.js` here (meh. I feel bad just writing this).

Reply:
Please.